README.md

```diff
@@ -40,12 +40,12 @@ and even better, on a video projector in your local hackspace!
 
 ## How it works
 
-Images are crawled from multiple public pre-configured sources.  
+Images are crawled from multiple public pre-configured sources.
 No image is actually downloaded, just the URL to images are gathered. Found images are kept in a list, also it is
 assured that the same URL is never gathered twice.
 
-To display the crawled images, _nichtparasoup_ starts a web-server display a web UI.  
```

Comment:
could you revert the removal of the trailing spaces.
they were intended for a line break